ncm-sysconfig/src/main/perl/sysconfig.pm

```diff
@@ -1,11 +1,51 @@
 #${PMcomponent}
 
+=head1 NAME
+
+sysconfig: management of sysconfig files
+
+=head1 DESCRIPTION
+
+The I<sysconfig> component manages system configuration files in
+C<< /etc/sysconfig >> . These are files which contain key-value pairs.
+However there is the possibility to add verbatim text either
+before or after the key-value pair definitions.
+
+=head1 FILE DEFINITIONS
+
+=over
+
+=item * C<< /software/components/sysconfig/files >>
+
+This is an dict which has the file name (unescaped) as the key, and
+the content information as the value. The value is an dict.
+
+=item * C<< /software/components/sysconfig/files/<fname>/ >>
+
+This is a dict containing key-value pairs. Both are strings.
+There are two special keys C<< prologue >> and C<< epilogue >> which contain
+text which will be copied verbatim into the file before or after
+the pair definitions, respectively.
+
+=back
+
+=head1 EXAMPLE
+
+    "/software/components/sysconfig/files/scfg" = dict(
```

Comment:
also move to schema (you can add a and `@example{}` after the `@documentation` of the main component type)

Reply:
Really? This causes `panc` to barf on the `"`

Reply:
It's happy with the example in the `@documentation` block, but not `@example`

Reply:
what is the error message? is it similar to https://github.com/quattor/pan/issues/127?
i'll try to find where i used it ...

Reply:
@jrha that's strange. i'm fine with it in the `@documenation` section, but i thought i already used `@example{}` somewhere....

Reply:
```
[INFO] --- panc-maven-plugin:10.2:check-syntax (check-generated-pan-syntax) @ sysconfig ---
parse error [/home/centos/configuration-modules-core/ncm-sysconfig/target/pan/components/sysconfig/schema.pan:24.1-34.1]
Encountered " <ERROR> "\' "" at line 2, column 1.
Was expecting one of:
    <EOF> 
    <KEY> ...
```

Reply:
hah, so it is similar to https://github.com/quattor/pan/issues/127
i made https://github.com/quattor/pan/issues/204 for followup